tests/guardrails/testNames.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
What is this used for?

Reply:
just to keep the naming consistent across all of the guardrails tests ... I would change the naming in one and then it be incorrect in the others